tables/Tables/TableCache.cc

```diff
@@ -170,9 +170,8 @@ void TableCache::flushTable (const String& name,
 PlainTable* TableCache::lookCache (const String& name, int tableOption,
                                    const TableLock& lockOptions)
 {
-    ScopedMutexLock sc(itsMutex);
```

Comment:
the mutex is not necessary for the full function, we only need it to access the shared state, the `tableMap_p` which is done in the operator call following it.
if we keep this mutex we would need recursive locking as the reopenRW will call isReadable which calls back into the cache.